app/components/Form/FormBase.tsx

```diff
@@ -6,7 +6,7 @@ import FormComponentProps from "./FormComponentProps";
 
 interface Props extends FormComponentProps {
   schema: JSONSchema7;
-  uiSchema: object;
+  uiSchema: { title: string };
```

Comment:
There is a [UiSchema type exported](https://github.com/rjsf-team/react-jsonschema-form/blob/76ab08c5b06b1cbecd03fab0bdbca64b69b5c038/packages/core/index.d.ts#L67). Granted, it doesn't do much in terms of type checking